test/functional/tests/volumes/test_discard_on_huge_core.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode
+from storage_devices.disk import DiskType, DiskTypeSet
+from core.test_run import TestRun
+from test_utils.size import Size, Unit
+
+mount_point = "/mnt/cas"
+syslog_path = "/var/log/messages"
```

Comment:
You won't find this log file in Debian based distros, because they log everything in /var/log/syslog. To be covered in such scenario, you can add a simple check:
```
from test_tools.fs_utils import check_if_file_exists
syslog_path = "/var/log/messages"
if not check_if_file_exists(syslog_path):
    syslog_path = "/var/log/syslog"
```

But... I don't see this log file used anywhere in this test, so you may get rid of it and just keep the above in mind for the future. ;)